app/models/manageiq/providers/vmware/infra_manager/vm_or_template_shared.rb

```diff
@@ -8,6 +8,10 @@ module ManageIQ::Providers::Vmware::InfraManager::VmOrTemplateShared
   include_concern 'RefreshOnScan'
   include_concern 'Scanning'
 
+  included do
+    supports :set_description
```

Comment:
I don't love `vm.set_description` as the method name, open to other options there